src/MCPServer/install/README.md

```diff
@@ -1,13 +1,135 @@
+    - **Description:** location of the rejected directory.
+    - **Type:** `string`
+    - **Default:** `"%%sharedPath%%rejected/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WAITONAUTOAPPROVE`**:
+    - **Description:** suspend execution of the current thread before attempting to approve a chain when it is pre-configured.
+    - **Type:** `int`
+    - **Default:** `"0"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORIESPOLLINTERVAL`**:
+    - **Description:** time in seconds between filesystem poll intervals .
+    - **Type:** `int`
+    - **Default:** `"1"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_LIMITTASKTHREADS`**:
+    - **Description:** max. number of threads that MCPServer will run simultaneously.
+    - **Type:** `int`
+    - **Default:** `"75"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_LIMITTASKTHREADSSLEEP`**:
+    - **Description:** suspend execution of the current thread for the given number of seconds before retrying to start a new thread when the limit is reached.
+    - **Type:** `float`
+    - **Default:** `"0.2"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_LIMITGEARMANCONNECTIONS`**:
+    - **Description:** max. number of concurrent connections to Gearman server.
+    - **Type:** `int`
+    - **Default:** `"10000"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_RESERVEDASTASKPROCESSINGTHREADS`**:
+    - **Description:** max. number of concurrent processing threads running foreground Gearman jobs.
+    - **Type:** `int`
+    - **Default:** `"8"`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_ENGINE`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `django.db.backends.mysql`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_DATABASE`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `MCP`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_USER`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `archivematica`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_PASSWORD`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `demo`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_HOST`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `localhost`
+
+- **`ARCHIVEMATICA_MCPSERVER_CLIENT_PORT`**:
+    - **Description:** a database setting. See [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) for more details.
+    - **Type:** `string`
+    - **Default:** `3306`
```

Comment:
I know this isn't part of this PR, but I think a comma should be inserted in the logging configuration sections of all of these READMEs. The string "If no changes are made to the new default configuration logs ..." should be "If no changes are made to the new default configuration, logs ...".